packages/cozy-stack-client/src/DocumentCollection.spec.js

```diff
@@ -512,6 +512,51 @@ describe('DocumentCollection', () => {
       expect(resp.data[0]).toHaveDocumentIdentity()
     })
 
+    it('should not throw an error if it is a missing index', async () => {
+      client.fetchJSON.mockRestore()
+      client.fetchJSON
+        .mockRejectedValueOnce(new Error('no_index'))
+        .mockReturnValueOnce(Promise.resolve({ rows: [] }))
+        .mockReturnValueOnce(Promise.resolve({}))
+        .mockReturnValueOnce(Promise.resolve(FIND_RESPONSE_FIXTURE))
+        .mockReturnValueOnce(Promise.resolve(FIND_RESPONSE_FIXTURE))
```

Comment:
FWIW, you can use mockResolvedValueOnce.

Reply:
*mind blown*